python/tank/authentication/resources/build_resources.sh

```diff
@@ -17,37 +17,64 @@ if [ -z "${PYTHON_BASE}" ]; then
     PYTHON_BASE="/Applications/Shotgun.app/Contents/Resources/Python"
 fi
 
-# Remove any problematic profiles from pngs.
-for f in *.png; do mogrify $f; done
-
 # Helper functions to build UI files
 function build_qt {
-    echo " > Building " $2
-
     # compile ui to python
+    echo "$1 $2 > $UI_PYTHON_PATH/$3.py"
     $1 $2 > $UI_PYTHON_PATH/$3.py
-
-    # replace PySide imports with local imports and remove line containing Created by date
-    sed -i "" -e "s/from PySide import/from .qt_abstraction import/g" -e "/# Created:/d" $UI_PYTHON_PATH/$3.py
+    # replace PySide2 imports with local imports and remove line containing Created by date
+    sed -i"" -E "/^from PySide2.QtWidgets(\s.*)?$/d; /^\s*$/d" $UI_PYTHON_PATH/$3.py
+    sed -i"" -E "s/^(from PySide.\.)(\w*)(.*)$/from .qt_abstraction import \2\nfor name, cls in \2.__dict__.items():\n    if isinstance(cls, type): globals()[name] = cls\n/g" $UI_PYTHON_PATH/$3.py
+    sed -i"" -E "s/from PySide2 import/from .qt_abstraction import/g" $UI_PYTHON_PATH/$3.py
```

Comment:
I see a lot more sed replacements here than in the example shotgunsoftware/tk-flame-review#17. Why is it different?


Also, if want to chain multiple sed replacement, it might be more elegant to apply this syntax:
```bash
   # replace PySide2 imports with local imports and remove line containing Created by date
sed -i"" \
  -E "/^from PySide2.QtWidgets(\s.*)?$/d; /^\s*$/d"
  -E "s/^(from PySide.\.)(\w*)(.*)$/from .qt_abstraction import \2\nfor name, cls in \2.__dict__.items():\n    if isinstance(cls, type): globals()[name] = cls\n/g" \
  -E "s/from PySide2 import/from .qt_abstraction import/g" \
  $UI_PYTHON_PATH/$3.py
```